UNL-CSE-AMBASSADORS/VITA
# Use EmailUtilities

Use the EmailUtilities utility class in all the places the`mail()` function was being used.

**TESTING**
Unless you have a mail server set up locally, you probably won't be able to test this in it's entirety. However, this one is fairly simple so hopefully it's not an issue
1. Ensure you can still log in
2. Ensure the password forgot/register still works
3. Ensure the cancelling a shift still works
